kythe/cxx/verifier/souffle_interpreter.h

```diff
@@ -17,8 +17,24 @@
 #ifndef KYTHE_CXX_VERIFIER_SOUFFLE_INTERPRETER_H_
 #define KYTHE_CXX_VERIFIER_SOUFFLE_INTERPRETER_H_
 
+#include "kythe/cxx/verifier/assertions.h"
+
 namespace kythe::verifier {
-bool RunSouffle();
+class Verifier;
```

Comment:
Is there a circular dependency? otherwise just include the header.

Reply:
Yes, with the :lib target.